Yafc/Workspace/ProductionTable/ProductionTableView.cs

```diff
@@ -401,8 +401,8 @@ private class IngredientsColumn(ProductionTableView view) : ProductionTableDataC
                 else {
                     for (int i = 0; i < recipe.recipe.ingredients.Length; i++) {
                         var ingredient = recipe.recipe.ingredients[i];
-                        var link = recipe.links.ingredients[i];
-                        var goods = recipe.links.ingredientGoods[i];
+                        var link = recipe.hierarchyEnabled ? recipe.links.ingredients[i] : null;
+                        var goods = recipe.hierarchyEnabled ? ingredient.goods : null;
```

Comment:
If that's okay with you, please add documentation to the fields `RecipeRow.enabled` and `RecipeRow.hierarchyEnabled`, so the reader can understand the difference between them without browsing their usage.